google/cloud/aiplatform/training_jobs.py

```diff
@@ -2502,6 +2503,8 @@ def run(
                 that training might stop before the entire training budget has been
                 used, if further training does no longer brings significant improvement
                 to the model.
+            additional_experiments (List[str]):
+                Optional. Experiment flags that can enable some experimental training features.
```

Comment:
For now, it will be regularly used by internal testing(see b/186151733), and we plan to use it to control the per-request feature rollout or fix, so there is no concrete list of all the flags now.

In the future, when we will update our doc on GCP to include some flags if the users can use them to disable/enable some features on per-request basis if the features doesn't fit in the use cases.